jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -92,13 +91,11 @@
     private final int _send;
     private static final int SEND_SERVER = 0x01;
     private static final int SEND_XPOWEREDBY = 0x02;
-    private static final Trie<Boolean> ASSUMED_CONTENT_METHODS = new ArrayTrie<>(8);
-
-    static
-    {
-        ASSUMED_CONTENT_METHODS.put(HttpMethod.POST.asString(), Boolean.TRUE);
-        ASSUMED_CONTENT_METHODS.put(HttpMethod.PUT.asString(), Boolean.TRUE);
-    }
+    private static final Index<Boolean> ASSUMED_CONTENT_METHODS = new Index.Builder<Boolean>()
```

Comment:
This could just be a map, as I think it really should be case sensitive (old mistake)... or even just a string switch in the one place it is used.   But happy to leave for another PR. your call.

Reply:
I think this should go into another PR as we don't want to delay 10.0.0 for too long.